.env

```diff
@@ -22,7 +23,10 @@ STAGE_WEB_GUEST_PORT=80
 STAGE_WEB_HOST_PORT=80
 STAGE_WEB_GUEST_SECURE_PORT=443
 STAGE_WEB_HOST_SECURE_PORT=443
+IS_STAGING=1
 
+PROD_CERTBOT_EMAIL="admin@devopsloft.io"
+PROD_APP_DOMAIN="devopsloft.io"
```

Comment:
the prod domain name is: 'www.devopsloft.io'. Does it matter?

Reply:
no, there is a DNS record that point to the site from devopsloft.io and from www.devopsloft.io

Reply:
Do we need to do some configurations in Route 53?

Reply:
yes,for the staging there should be a sub domain record that point to the staging server
in order to check that the certificate signing works well

Reply:
In that case, I would need your help on this one.  Can you join the coming office hours meetup?

Reply:
i will try to come to this week meetup